<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>300</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QCheckBox" name="compilerRuntimeCheck">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>151</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>包含编译器运行时</string>
   </property>
  </widget>
  <widget class="QPushButton" name="deployButton">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>110</y>
     <width>101</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>开始部署</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="noTranslationsCheck">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>180</y>
     <width>151</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>不包含翻译文件</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="noSystemD3DCheck">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>210</y>
     <width>151</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>不包含系统D3D编译器</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="noOpenglSwCheck">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>240</y>
     <width>151</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>不包含OpenGL软件实现</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="verboseCheck">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>270</y>
     <width>151</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>显示详细输出</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="logTextEdit">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>150</y>
     <width>231</width>
     <height>141</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="statusLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>281</width>
     <height>21</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="outputDirEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>281</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="exePathEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>281</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="browseExeButton">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>30</y>
     <width>101</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>浏览…</string>
   </property>
  </widget>
  <widget class="QPushButton" name="browseOutputButton">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>70</y>
     <width>101</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>指定输出目录</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
